src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -194,7 +194,7 @@ private HttpVariableDeclarationNode ParseVariableDeclaration()
 
                 while (MoreTokens())
                 {
-                    if (CurrentToken is { Text: "@" } or { Kind: HttpTokenKind.Word } or { Text: "_" })
+                    if (CurrentToken is { Text: "@" } or { Kind: HttpTokenKind.Word } or { Text: "_" } or {Text: "." })
```

Comment:
```suggestion
                    if (CurrentToken is { Kind: HttpTokenKind.Word } or { Text: "@" or "_" or "." })
```

Reply:
It looks like this might also allow variable names with `@` in the middle of the word. Is this intended?